components/eventing-controller/pkg/handlers/beb.go

```diff
@@ -0,0 +1,203 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+
+	eventingv1alpha1 "github.com/kyma-project/kyma/components/eventing-controller/api/v1alpha1"
+	client2 "github.com/kyma-project/kyma/components/eventing-controller/pkg/ems2/api/events/client"
```

Comment:
where do the '2' come from? i dont see client1, config1, types1, auth1

Reply:
Only as a convention, we have used it in the past inside of ems2 library to mark that the objects are part of this library

Reply:
Refactored, renaming the old names "*2" to "*"